common/Log.hpp

```diff
@@ -294,6 +294,17 @@ static constexpr std::size_t skipPathPrefix(const char (&s)[N], std::size_t n =
         LOG.flush();                                                                               \
     } while (false)
 
+#define LOG_BODY__(LVL, X, TYPE, PREFIX, SUFFIX)                                                   \
```

Comment:
Two macros that differ by the number of underscores in the name is downright unfriendly. 

Can we find a more sensible name please? Perhaps `LOG_BODY_IMPL` or something like that ?

Reply:
Sure, I will look another name